android/app/src/androidTest/java/org/chrome/ExampleInstrumentedTest.java

```diff
@@ -22,7 +22,7 @@
 import static org.junit.Assert.*;
 
 /**
- * Instrumented test, which will execute on an Android device.
+ * Instrumented test, which will execute on an Android org.chrome.device.
```

Comment:
dont think this test is necessary - but if you are going to check this in, remove the "org.chrome." from here

Reply:
I'll remove it